apps/cowswap-frontend/src/common/hooks/useOnSelectNetwork.ts

```diff
@@ -8,7 +8,7 @@ import { ApplicationModal } from 'legacy/state/application/reducer'
 import { updateConnectionError } from 'legacy/state/connection/reducer'
 import { useAppDispatch } from 'legacy/state/hooks'
 
-import { getWeb3ReactConnection } from 'modules/wallet/web3-react/connection'
```

Comment:
Because im only starting the review, and i don't know where this is ending, my comment might get outdated soon, but one thought here is that long term we shouldn't export something Web3React related (if we can avoid it).

Ideally the module would abstract that

Reply:
100% agree. In this PR I only moved all imports from `modules/wallet/*` to `modules/wallet` to replace it by import from a in in the future